CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.58.0      2024-08-27
+
+* Log Insights:
+  - New log parsing mechanism that is more performant and allows
+    for arbitrary log_line_prefix settings. The new parsing mechanism
+    is the default, but you can set `db_log_line_prefix = legacy` in
+    the config file or `DB_LOG_LINE_PREFIX=legacy` in the environment
```

Comment:
Turns out the env part of this does not work yet: I missed that in #494. PR coming up.